modules/virt-aws-bm.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-aws-bm_{context}"]
+= {VirtProductName} on AWS bare metal
+
+You can run {VirtProductName} on an Amazon Web Services (AWS) bare metal {product-title} cluster. Before you set up your cluster, review the following summary of supported features and limitations:
+
+Installing::
+
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare metal node instance types in the `install-config.yaml` file. For example, you can use the `c6g.metal` type value for a machine based on 64-bit ARM architecture.
```

Comment:
ARM of only in Dev Preview. We should mention another metal instance here.

Reply:
for example we are testing with `c5n.metal` which is an x86_64 architecture.

Reply:
Ah, thank you for that clarification. I'll swap it shortly